spec/features/admin/subscriptions_spec.rb

```diff
@@ -224,17 +224,42 @@ def open_subscription_products_panel
         select2_select "Australia", from: "bill_address_country_id"
         select2_select "Victoria", from: "bill_address_state_id"
 
+        # Making sure that once set with empty it actually changes, and add the phone back
+        fill_in "bill_address_phone", with: ''
+        expect(page).to have_input "bill_address_phone", with: ''
+        expect(page).to have_content 'can\'t be blank', count: 1
+        fill_in "bill_address_phone", with: '0400 123 456'
+
         # Use copy button to fill in ship address
         click_link "Copy"
         expect(page).to have_input "ship_address_firstname", with: 'Freda'
         expect(page).to have_input "ship_address_lastname", with: 'Figapple'
         expect(page).to have_input "ship_address_address1", with: '7 Tempany Lane'
+        expect(page).to have_input "ship_address_city", with: 'Natte Yallock'
+        expect(page).to have_input "ship_address_zipcode", with: '3465'
+        expect(page).to have_input "ship_address_phone", with: '0400 123 456'
+        expect(page).to have_input "ship_address_country_id", with: 'Australia'
+        expect(page).to have_input "ship_address_state_id", with: 'Victoria'
```

Comment:
I see your second commit removes these assertions. But I think it might be a good idea to check them as you did initially, only using a different matcher: `ship_address_country_id` and `ship_address_state_id` are selectors and input not fields, so I think `have_input` is triggering the error. One idea could be trying `have_selector` instead:
`expect(page).to have_selector '<dropdown_identifyer>', text: '<dropdown_option>'`